casper/src/test/scala/coop/rchain/casper/api/BlockQueryResponseAPITest.scala

```diff
@@ -146,4 +175,45 @@ class BlockQueryResponseAPITest extends FlatSpec with Matchers with BlockStoreFi
       s"Error: Failure to find block with hash ${badTestHashQuery}"
     )
   }
+
+  "findBlockWithDeploy" should "return error when no block matching query exists" in withStore {
```

Comment:
I would group `findBlockWithDeploy` tests together, one after the other, rather than interleaving them with `showBlocks` tests.

Reply:
Done